Bot.py

```diff
@@ -123,9 +137,9 @@ def _postTweet(self, call: dict) -> None:
         # Check for weird old calls
         if abs(diff.total_seconds()) >= 1.8e3:
             return
-        elif call["len"] < self.callThreshold:
+        elif call["len"] < self.config["general"]["call_threshold"]:
```

Comment:
Do you think it'd make sense to construct a config object or NamedTuple with these values as attributes, rather than having to access into the dictionary several levels every time we need a value?